hudi-flink/src/main/java/org/apache/hudi/streamer/HoodieFlinkStreamer.java

```diff
@@ -97,8 +99,21 @@ public static void main(String[] args) throws Exception {
       }
     }
 
-    DataStream<HoodieRecord> hoodieRecordDataStream = Pipelines.bootstrap(conf, rowType, parallelism, dataStream);
+    // bulk_insert mode and Append mode is same in streaming
+    final String writeOperation = conf.get(FlinkOptions.OPERATION);
+    if (WriteOperationType.fromValue(writeOperation) == WriteOperationType.BULK_INSERT
+        || OptionsResolver.isAppendMode(conf)) {
+      Pipelines.append(conf, rowType, dataStream);
+    }
+
```

Comment:
Do we still need bootstrap for `bulk_insert` and `append` mode ?

Reply:
This is my negligence, it has been fixed.